.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: deploy
+
+on:
+  push:
+    branches: [master]
```

Comment:
I removed the `pull_request` from `on` because, unless I'm misunderstanding what GitHub is saying, it was deploying the app every time a pull request to `master` was created:

<img width="695" alt="screenshot" src="https://user-images.githubusercontent.com/3039310/84907911-cb854500-b081-11ea-8c0e-019478efc710.png">

Reply:
ok, yes that makes sense.